update deployment YAML files (#645)
accuknox/KubeArmor: contribution/testing_guide.md

# Testing Guide

> There are two ways to check the functionalities of KubeArmor: 1) testing kubeArmor in manual and 2) using the testing framework.

Reviewer: ```suggestion
There are two ways to check the functionalities of KubeArmor: 1) testing kubeArmor manually and 2) using the testing framework.
```

> # 1.  Test KubeArmor in manual

Reviewer: ```suggestion
# 1.  Test KubeArmor manually
```

        --msgPath string     Output location for messages, {path|stdout|none} (default "none")
    ```
    

>     Note that you will see alerts and logs generated right after `karmor` runs logs; thus, we recommend to run the above command in other terminal to see logs live and in an interactive way to avoid any kind of interference.

Reviewer: ```suggestion
    Note that you will see alerts and logs generated right after `karmor` runs logs; thus, we recommend to run the above command in other terminal to see logs live.
```

    ~/KubeArmor/tests$ cat /tmp/kubearmor.test
    ```

- The case that KubeArmor is running as a daemonset in Kubernetes

>     Run the auto-testing framework

Reviewer: ```suggestion
    Run the testing framework
```
